opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/async/AsyncTestConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package io.opentracing.contrib.spring.cloud.async;
+
+import feign.Feign;
+import io.opentracing.Tracer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.util.concurrent.Executor;
+
+import static org.mockito.Mockito.mock;
+
+@Configuration
+@EnableAsync
+@ComponentScan("io.opentracing.contrib.spring.cloud.async")
```

Comment:
just to make the test bit efficient, and confined only to Async testing, otherwise it tends to load a whole lot from classpath which is not required for this test context